app/views/groups/index.html.erb

```diff
@@ -9,10 +23,23 @@
 
   <tbody>
     <% @groups.each do |group| %>
-      <tr>
-        <td><%= group.id %></td>
+      <tr class="<%= group.neighborhood.id %> <%= group.kind %>">
```

Comment:
Hah! Silly me. This one was caused by me not re-seeding the database, since you updated seeds.rb. The solution is to `rake db:drop`, then `rake: db:reset`, and restart the server @HacksForHugs